src/GNDStk/Node/test/all.test.cpp

```diff
@@ -18,8 +18,10 @@ SCENARIO("Testing GNDStk Node all()") {
          // nodes: chemicalElement
          bool found = false;
          auto elem = elems.all("chemicalElement",found);
```

Comment:
Is passing `found` here required? This feels a lot like C syntax, which makes me sad.

Reply:
Don't let it make you sad. GNDStk's `found` out-parameter is always optional. If you don't provide it, the behavior is like that of other codes in NJOY21: throw an exception if there's a problem. If you do send `found`, then the behavior is a bit different: the function will set `found` to `true` or `false` as appropriate, but *not* throw an error if it wasn't found. This would be convenient for a user who knows that something might or might not be found, is prepared to handle either case, and doesn't want the extra clutter of putting `try/catch` material in their own code.